pkg/networkmanager/wireguard.jsx

```diff
@@ -0,0 +1,173 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { Name, NetworkModal, dialogSave } from "./dialogs-common";
+import cockpit from 'cockpit';
+import { Button } from '@patternfly/react-core/dist/esm/components/Button/index.js';
+import { ClipboardCopy } from '@patternfly/react-core/dist/esm/components/ClipboardCopy/index.js';
+import { FormGroup, FormFieldGroup, FormFieldGroupHeader } from '@patternfly/react-core/dist/esm/components/Form/index.js';
+import { InputGroup } from '@patternfly/react-core/dist/esm/components/InputGroup/index.js';
+import { TextInput } from '@patternfly/react-core/dist/esm/components/TextInput/index.js';
+import { EmptyState, EmptyStateBody } from '@patternfly/react-core/dist/esm/components/EmptyState/index.js';
+import { Grid } from '@patternfly/react-core/dist/esm/layouts/Grid/index.js';
+import { ModelContext } from './model-context';
+import { useDialogs } from 'dialogs.jsx';
+import { MinusIcon } from '@patternfly/react-icons';
+
+import './wireguard.scss';
+
+const _ = cockpit.gettext;
+
+export function WireGuardDialog({ settings }) {
+    const idPrefix = 'network-wireguard-settings';
+    const model = useContext(ModelContext);
+    const Dialogs = useDialogs();
+    console.log(model);
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230626-161008-14729dcc-fedora-38-devel-networking//Coverage/lcov/github-pr.diff.gcov.html)